src/extensions/default/bramble/stylesheets/sidebarTheme.css

```diff
@@ -1,68 +1,250 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:400,600');
```

Comment:
What does this do to our load times?  Probably it's fine, but we've had font loading issues in the past.  Do we load this font elsewhere, and will it be cached?